src/07-projects/milestones.apib

```diff
@@ -89,6 +93,10 @@ Get details for a single milestone.
                     + time_and_materials
                     + fixed_price
                     + non_invoiceable
+            + `budget` (object) - This is only available for users that have access to budgets on projects.
+                + `allocated` (Money)
```

Comment:
as discussed in person, I now think that "allocated" actually creates confusion around "is this allocated *to* the project, or allocated *within* the project"...

_provided_ budget might seem to be a good alternative...

Reply:
Isn't `allocated` also the estimated budget?

Reply:
Also, if it's between allocated and provided I would choose allocated.

Reply:
After some research we think "provided budget" for a milestone (so when you create a milestone you specify that budget) is better than "allocated". Allocated in this case means the budget you plan to spend, so haven't spent yet.